docs: add from which version public_metrics start (#671)
redpanda-data/documentation: docs/cluster-administration/monitoring.mdx

    <meta name="description" content="Metrics to monitor the health of your system to predict issues and optimize performance."/>
</head>

> You can monitor the health of your system to optimize performance. The most useful metrics are available on `<node ip>:9644/public_metrics`, which are available in Redpanda since v22.2. This section describes how to use Prometheus queries to monitor your system, for example, to check consumer group lag or Kafka producer request latency.

Reviewer: ```suggestion
You can monitor the health of your system to optimize performance. The most useful metrics are available on `<node ip>:9644/public_metrics`, which has been available since v22.2. This section describes how to use Prometheus queries to monitor your system (for example, to check consumer group lag or Kafka producer request latency).
```

> You can monitor the health of your system to optimize performance. The most useful metrics are available on `<node ip>:9644/public_metrics`, which are available in Redpanda since `v22.2`. This section describes how to use Prometheus queries to monitor your system, for example, to check consumer group lag or Kafka producer request latency.

Author: You can monitor the health of your system to optimize performance. The most useful metrics are available on `<node ip>:9644/public_metrics`, which are available in Redpanda since v22.2. This section describes how to use Prometheus queries to monitor your system, for example, to check consumer group lag or Kafka producer request latency.
```suggestion
You can monitor the health of your system to optimize performance. The most useful metrics are available on `<node ip>:9644/public_metrics`, which are available in Redpanda since `v22.2`. This section describes how to use Prometheus queries to monitor your system, for example, to check consumer group lag or Kafka producer request latency.
```
